### PR TITLE
doc: add module_timeouts to terradoc

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

**Review by katcipis — DISMISSED** [review]
LGTM + assuming generating the README.md from the tfdoc is done in a separate step ?

**Review by mariux — CHANGES_REQUESTED** [review]
please regenerate readme

File: README.tfdoc.hcl [hunk 1]
```diff
@@ -95,6 +95,23 @@ section {
           END
         }
 
+        variable "module_timeouts" {
+          type           = any
+          readme_type    = "object(resource_name)"
```
Comment (mariux) [inline README.tfdoc.hcl:100]:
```suggestion
          readme_type    = "map(timeouts)"
```

File: README.tfdoc.hcl [hunk 2]
```diff
@@ -95,6 +95,23 @@ section {
...
+          description    = <<-END
+            How long certain operations (per resource type) are allowed to take before being considered to have failed.
```
Comment (mariux) [inline README.tfdoc.hcl:102]:
```suggestion
            A map of timeout objects that is keyed by Terraform resource name defining timeouts for `create`, `update` and `delete` Terraform operations.
            Supported resource names are: `google_artifact_registry_repository`.
```

**Review by mariux — CHANGES_REQUESTED** [review]

File: README.tfdoc.hcl (hunk 1 above)
Comment (soerenmartius) [reply README.tfdoc.hcl:100]: this is wrong in the template as well in this case

**Comment by soerenmartius** [comment]
updated the generated README also

**Review by katcipis — APPROVED** [review]
LGTM + imagining that removing the terradoc target on make is part of some refactoring/fix

**Comment by soerenmartius** [comment]
> LGTM + imagining that removing the terradoc target on make is part of some refactoring/fix

yip - we had the same target duplicated in there :)

**Pull request merged**